src/models/Player.js

```diff
@@ -70,4 +83,4 @@ class Player {
   }
 }
 
-module.exports = { Player };
+module.exports = { Player };
```

Comment:
unrelated change?

Reply:
This is unrelated, really unsure why this happened or is changed.